libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/DBSchemaAccessor.java

```diff
@@ -81,6 +81,20 @@ default List<String> showTables(String schemaName) {
      */
     List<DBObjectIdentity> listTables(String schemaName, String tableNameLike);
 
+    /**
+     * Show all external table names list in the specified schema
+     */
+    default List<String> showExternalTables(String schemaName) {
```

Comment:
```java

    /**
     * Get all table columns in the specified schema
     */
    Map<String, List<DBTableColumn>> listTableColumns(String schemaName, List<String> tableNames);

    /**
     * Get all table columns in the specified schema and table
     */
    List<DBTableColumn> listTableColumns(String schemeName, String tableName);

    /**
     * Get all table columns(hold only basic info) in the specified schema
     */
    Map<String, List<DBTableColumn>> listBasicTableColumns(String schemaName);

    /**
     * Get all table columns(hold only basic info) in the specified schema and table
     */
    List<DBTableColumn> listBasicTableColumns(String schemaName, String tableName);
```

should we add these methods for external table?

Reply:
The implementation of showtables is the default check basic_table, so I wrote it out separately, and other can be reused